tests/acceptance/features/apiNotification/emailNotification.feature

```diff
@@ -9,6 +9,7 @@ Feature: Email notification
       | username |
       | Alice    |
       | Brian    |
+      | Carol    |
```

Comment:
As we don't need this user for all the scenarios, it's not suitable to make it in the background. Please make this user in the scenario where it is needed
```suggestion
```